2006Redone Server/src/redone/net/packets/impl/Bank10.java

```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
 			player.getPlayerAssistant().useOperate(removeId);
 			break;
 		case 3900:
-			player.getShopAssistant().buyItem(removeId, removeSlot, 5);
+			player.getShopAssistant().buyItem(removeId, removeSlot, 10);
 			break;
 
 		case 3823:
 			if(!player.getItemAssistant().playerHasItem(removeId)) {
 				return;
 			}
-			player.getShopAssistant().sellItem(removeId, removeSlot, 5);
+			player.getShopAssistant().sellItem(removeId, removeSlot, 10);
+			player.getItemAssistant().resetItems(3823);

```

Comment:
Doesn't look like this line is needed?

Reply:
Indeed the resetItems is not needed, i should have tested it before. 
The bank method used it and first it was not working (check gifs in discord) so i added it.